src/components/DropDown/DropDown.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Menu, Dropdown } from 'antd';
+import { DownOutlined } from '@ant-design/icons';
+import { NavLink } from 'react-router-dom';
+import { Avatar } from 'antd';
+
+import { UserOutlined } from '@ant-design/icons';
+
+interface DropDownProps {}
+
+const menu = (
+  <Menu>
+    <Menu.Item key="0">
+      <NavLink className="activeNot" to="/login">
+        Log in
+      </NavLink>
+    </Menu.Item>
+    <Menu.Item key="1">
+      <a href="#">2nd menu item</a>
+    </Menu.Item>
+    <Menu.Divider />
+    <Menu.Item key="3">Log out</Menu.Item>
+  </Menu>
+);
```

Comment:
extract this into a separate component

Reply:
am mutat in componenta separata, dar isi pierde style-ul din antd. recomandarea de la antd e sa fie in aceeasi componenta, cum e mai sus